experimental/sample_web_static/device_multithreaded.c

```diff
@@ -33,8 +35,13 @@ iree_status_t create_device_with_static_loader(iree_allocator_t host_allocator,
   iree_host_size_t worker_local_memory = 0;
   iree_task_topology_t topology;
   iree_task_topology_initialize(&topology);
-  // TODO(scotttodd): Try with more threads
-  iree_task_topology_initialize_from_group_count(/*group_count=*/1, &topology);
+  iree_task_topology_initialize_from_group_count(
+      /*group_count=*/4, &topology);
+  // Note: threads increase memory usage. If using a high thread count, consider
+  // passing in a larger WebAssembly.Memory object, increasing Emscripten's
+  // INITIAL_MEMORY, or setting Emscripten's ALLOW_MEMORY_GROWTH.
+  // iree_task_topology_initialize_from_group_count(
+  //     /*group_count=*/emscripten_num_logical_cores(), &topology);
```

Comment:
that's a lot of threads :P

![image](https://user-images.githubusercontent.com/4010439/153469567-af7502f9-2325-4da9-b244-06f94466a708.png)

Reply:
did you see actual wasm heap memory count increase or just browser increase? our cost per thread should be ~16-64KB (stack size + a few hundred bytes of overhead), so it'd be interesting to see if that was emscripten doing silly things with reserving MB stacks or something

Reply:
I'm not sure yet. Each Wasm page is 64KB (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/WebAssembly/Memory). I can check how many Emscripten defaults to creating and how many we actually intend to use.